projecting-collect.rst

```diff
@@ -239,7 +240,70 @@ If the command is successfully executed, you will find your file in the launcher
 .. image:: /img/project-collect/collect-emulator2.*
   :alt: Image showing collect app on the emulator screen
   
+.. _using-command-line:
+  
+Using Command Line 
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also run the emulator using command line. Follow the steps given below to start your emulator using the command line:
+
+.. note::
+
+  If SDK installation has been put in another drive or folder instead of in its default location of ``$USER_HOME`` or ``$HOME``. Make sure you have set the environment variables according to that. In the command line type the following command to set environment variables.
+  
+  .. code-block:: console
+  
+    set ANDROID_SDK_ROOT=path\sdk\
+
+
+1. Open the terminal and move to the :file:`emulator` folder of the `SDK` directory.
+
+.. code-block:: console
+
+  $ cd emulator
+  
+2. For the list of available virtual devices, type the following command:
+
+.. code-block:: console
+
+  $ emulator -list-avds
+ 
+.. tip::
+
+  If you are not able to locate :file:`emulator.exe` file in :file:`SDK` folder. Type the following command to know the location of the file:
+  
+  .. code-block:: console
+
+    $ which emulator
+  
+  On Windows:
+  
+  .. code-block:: doscon
+
+    > where emulator
+
+3. Use :command:`emulator` to start the emulator. Here *avd_name* is the name of Android virtual device that you have created.
+
+.. code-block:: console
+
+  emulator -avd avd_name
+
+.. note::
+  
+  - You can use :command:`sdkmanager` command to update, install, and uninstall packages for the Android SDK. This method is not recommended as it is not user-friendly and also takes time.
+
+  .. code-block:: console
+
+   sdkmanager packages [options]
+   (e.g sdkmanager --verbose "system-images;android-19;google_apis;x86")
+  
+  - To create and manage Android Virtual device from the command line, you can use :command:`avdmanager`.
+  
+  .. code-block:: console
+
+   avdmanager [global options] command [command options]
+   (e.g avdmanager -v create avd --name testAVD -k "system-images;android-19;google_apis;x86" -g "google_apis")
```

Comment:
> e.g. 
http://docs.opendatakit.org/style-guide/#avoid-latin

It isn't clear that you are presenting an abstract model for the command and then an example. Either present just the concrete form (and explain what options you are using), or explain what you are doing with a little more prose.

Also, terminal commands should be prefaced with the `$` symbol.